scripts/deploy_teleporter.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+set -e
+
+TELEPORTER_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")"
+  cd .. && pwd
+)
+
+if ! command -v forge &> /dev/null; then
+    echo "forge not found, installing"
+    $TELEPORTER_PATH/scripts/install_foundry.sh
+fi
+
+function printHelp() {
+    echo "Usage: ./scripts/deploy_teleporter.sh [OPTIONS]"
+    echo "Deploys a selected TeleporterMessenger contract to the specified chain"
+    echo "For a list of releases, go to https://github.com/ava-labs/teleporter/releases"
+    printUsage
+}
+
+function printUsage() {
+    echo "Options:"
+    echo "  --fund-deployer <private_key>    Optional. Funds the deployer address with the account held by <private_key>"
+    echo "  --version <version>              Required. Specify the release version to deploy"
+    echo "  --rpc-url <url>                  Required. Specify the rpc url of the node to use"
+    echo "  --help                           Print this help message"
+}
+
+teleporter_version=
+user_private_key=
+rpc_url=
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --fund-deployer) 
+            if [[ $2 != --* ]]; then
+                user_private_key=$2
+            else 
+                echo "Invalid private key $2" && printHelp && exit 1
+            fi 
+            shift;;
+        --version)  
+            if [[ $2 != --* ]]; then
+                teleporter_version=$2
+            else 
+                echo "Invalid teleporter version $2" && printHelp && exit 1
+            fi 
+            shift;;
+        --rpc-url)  
+            if [[ $2 != --* ]]; then
+                rpc_url=$2
+            else 
+                echo "Invalid rpc url $2" && printHelp && exit 1
+            fi 
+            shift;;
+        --help) 
+            printHelp && exit 0 ;;
+        *) 
+          echo "Invalid option: -$1" && printHelp && exit 1;;
+    esac
+    shift
+done
+
+# Tokens required to deploy the contract.
+# Equal to contractCreationGasLimit * contractCreationGasPrice
+# from utils/deployment-utils/deployment_utils.go
+gas_tokens_required=10000000000000000000 # 10^19 gwei
```

Comment:
Can this be shortened to `10ether`?

Reply:
It can, but `cast` returns balances in wei, so in order to compare them to check if we should fund or not, we need to keep it in wei